templates/admin/user/edit.tmpl

```diff
@@ -4,7 +4,7 @@
 			{{.locale.Tr "admin.users.edit_account"}}
 		</h4>
 		<div class="ui attached segment">
-			<form class="ui form" action="{{.Link}}" method="post">
+			<form class="ui form" action="./edit" method="post">
```

Comment:
But isn't that the same as
```suggestion
			<form class="ui form" action="edit" method="post">
```
?
Or are you explicitly doing it to mark it as a path?

Reply:
```      
m.Combo("/{userid}/edit").Get(admin.EditUser).Post(web.Bind(forms.AdminEditUserForm{}), admin.EditUserPost)
      m.Post("/{userid}/delete", admin.DeleteUser)
      m.Post("/{userid}/avatar", web.Bind(forms.AvatarForm{}), admin.AvatarPost)
      m.Post("/{userid}/avatar/delete", admin.DeleteAvatar)
```
Indicates explicitly that the requested path is in the same directory as the current page.